swarm_copy/app/database/db_utils.py

```diff
@@ -0,0 +1,45 @@
+"""Utilities for the agent's database."""
+
+import json
+from typing import Any
+
+from sqlalchemy import select
+from sqlalchemy.ext.asyncio import AsyncSession
+
+from swarm_copy.app.database.sql_schemas import Messages, Threads, utc_now
+
+
+async def put_messages_in_db(
+    history: list[dict[str, Any]], thread_id: str, offset: int, session: AsyncSession
+) -> None:
+    """Add the new messages in the database."""
+    for i, messages in enumerate(history):
+        new_msg = Messages(
+            order=i + offset,
+            thread_id=thread_id,
+            content=json.dumps(messages),
+        )
+        session.add(new_msg)
+    # we need to update the thread update time
+    thread = await session.get(Threads, thread_id)
```

Comment:
I suggest we throw an error here, if the thread could not be retrieved from the thread_id we're in big trouble (i.e. messages are being assigned to an ghost thread)